mk/config.mk

```diff
@@ -38,8 +38,10 @@ WARNS ?= 3
 # Define DEBUG=1 to compile with -g option
 # DEBUG=1
 
-# If 1, debug mode of the tee firmware (CPU restart, Core Status)
-CFG_TEE_CORE_DEBUG ?= 0
+# Core debug mode enablement
+# When defined to "y", tee firmware is in debug mode. Core and pager status,
+# panics verbosity, physical/virtual address paranoid verifications.
+CFG_TEE_CORE_DEBUG ?= n
```

Comment:
ok

On 11 July 2016 at 10:20, Jens Wiklander <notifications@github.com> wrote:

> In mk/config.mk
> <https://github.com/OP-TEE/optee_os/pull/909#discussion_r70217698>:
>
> > @@ -38,8 +38,10 @@ WARNS ?= 3
> >  # Define DEBUG=1 to compile with -g option
> >  # DEBUG=1
> >
> > -# If 1, debug mode of the tee firmware (CPU restart, Core Status)
> > -CFG_TEE_CORE_DEBUG ?= 0
> > +# Core debug mode enablement
> > +# When defined to "y", tee firmware is in debug mode. Core and pager status,
> > +# panics verbosity, physical/virtual address paranoid verifications.
> > +CFG_TEE_CORE_DEBUG ?= n
>
> I wouldn't mind having that changed. But it would be better to take in a
> separate PR which I assume could be merged before this PR.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/OP-TEE/optee_os/pull/909/files/8ba67ef14cc23de88cb7bba06a4cefffb775401a#r70217698>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ATEHJnuhrlK6NGBxt6sxg3yhajHWxIZ2ks5qUfzkgaJpZM4JIk9V>
> .
>